specs/core/1_custody-game.md

```diff
@@ -689,28 +699,6 @@ Append this to `process_final_updates(state)`:
     )
     for index, validator in enumerate(state.validator_registry):
         if index not in validator_indices_in_records:
-            if validator.exit_epoch != FAR_FUTURE_EPOCH and validator.withdrawable_epoch == FAR_FUTURE_EPOCH:
+            if validator.exit_epoch != FAR_FUTURE_EPOCH:
```

Comment:
```suggestion
            if validator.exit_epoch != FAR_FUTURE_EPOCH and validator.withdrawable_epoch == FAR_FUTURE_EPOCH:
```

I don't think this line should be changed. The problem with removing `and validator.withdrawable_epoch == FAR_FUTURE_EPOCH` is that an exited validator's withdrawal epoch keeps being pushed out each epoch.

Reply:
You're right, I'm dizzy now. 😵
There's only suspense, not delay in `process_*_challenge` functions.